project_e/dealers/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.urls import reverse
+#from hashid_field import HashidAutoField
+
+# Create your models here.
+class Dealer(models.Model):
+    ref_id = models.AutoField(primary_key=True)
```

Comment:
Is this overwriting the current dealer app? Im confused why this is showing a fresh change when dealer app is exisitng

Reply:
I'd presume because I changed ref_id to an autofield instead of HashID? It's the same Dealer app